worker/multi_tenancy.go

```diff
@@ -0,0 +1,42 @@
+// +build oss
+
+/*
+ * Copyright 2021 Dgraph Labs, Inc. and Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package worker
+
+import (
+	"context"
+
+	"github.com/dgraph-io/dgraph/protos/pb"
+)
+
+func (w *grpcWorker) DeleteNamespace(ctx context.Context,
+	req *pb.DeleteNsRequest) (*pb.Status, error) {
+	return nil, nil
+}
```

Comment:
Shouldn't we return an error here, like `ErrNotSupported`? (see Backup)

Reply:
Done